copper-server-api/src/main/java/com/pogeyan/cmis/api/data/IObjectFlowService.java

```diff
@@ -1,12 +1,28 @@
 package com.pogeyan.cmis.api.data;
 
+import java.util.List;
 import java.util.Map;
 
 import javax.management.modelmbean.InvalidTargetObjectTypeException;
 
+import org.apache.chemistry.opencmis.commons.data.Acl;
+import org.apache.chemistry.opencmis.commons.data.Properties;
+import org.apache.chemistry.opencmis.commons.enums.UnfileObject;
+import org.apache.chemistry.opencmis.commons.server.ObjectInfoHandler;
+import org.apache.chemistry.opencmis.commons.spi.Holder;
+
 public interface IObjectFlowService {
 	public void setObjectFlowStoreSettings(IObjectFlowStoreSetting dbSettings) throws InvalidTargetObjectTypeException;
 
+	public Boolean beforeCreation(String repositoryId, String folderId, Properties properties, List<String> policies,
```

Comment:
rename folderId -> objectId as we need to keep this API generic. Can we also change from Boolean -> bool? so it uses the base type.